static/app/views/performance/browser/webVitals/pagePerformanceTables.tsx

```diff
@@ -52,25 +54,17 @@ export function PagePerformanceTables() {
     .slice(0, MAX_ROWS);
   const bad = tableData.filter(row => row.score < 50).slice(0, MAX_ROWS);
 
-  const getRenderHeadCell = (label, threshold) => {
-    return function (col: Column) {
-      if (col.key === 'score') {
-        return <AlignRight>{threshold}</AlignRight>;
-      }
-      if (col.key === 'transaction') {
-        return <NoOverflow>{label}</NoOverflow>;
-      }
-      return <AlignRight>{col.name}</AlignRight>;
```

Comment:
<img width="400" alt="Screenshot 2023-09-21 at 12 44 09 PM" src="https://github.com/getsentry/sentry/assets/63818634/8ced1007-24e1-4e33-9b63-7f36343c57e0">

it's kind of unclear what the second column represents without the header, maybe something to take up with dora?

Reply:
yeah its supposed to be the count/# of page loads. Thanks for the feedback, ill bring it up!